File: tools/recovery/fern_vm_all/TeamCompassViewModel$special$$inlined$CoroutineExceptionHandler$1.java
package com.example.teamcompass.ui;

import android.util.Log;
import kotlin.Metadata;
import kotlin.coroutines.AbstractCoroutineContextElement;
import kotlin.coroutines.CoroutineContext;
import kotlin.jvm.internal.SourceDebugExtension;
import kotlinx.coroutines.CoroutineExceptionHandler;

@Metadata(
   mv = {2, 0, 0},
   k = 1,
   xi = 48,
   d1 = {"\u0000!\n\u0000\n\u0002\u0018\u0002\n\u0002\u0018\u0002\n\u0000\n\u0002\u0010\u0002\n\u0000\n\u0002\u0018\u0002\n\u0000\n\u0002\u0010\u0003\n\u0000*\u0001\u0000\b\n\u0018\u00002\u00020\u00012\u00020\u0002J\u0018\u0010\u0003\u001a\u00020\u00042\u0006\u0010\u0005\u001a\u00020\u00062\u0006\u0010\u0007\u001a\u00020\bH\u0016¨\u0006\t¸\u0006\u0000"},
   d2 = {"kotlinx/coroutines/CoroutineExceptionHandlerKt$CoroutineExceptionHandler$1", "Lkotlin/coroutines/AbstractCoroutineContextElement;", "Lkotlinx/coroutines/CoroutineExceptionHandler;", "handleException", "", "context", "Lkotlin/coroutines/CoroutineContext;", "exception", "", "kotlinx-coroutines-core"}
)
@SourceDebugExtension({"SMAP\nCoroutineExceptionHandler.kt\nKotlin\n*S Kotlin\n*F\n+ 1 CoroutineExceptionHandler.kt\nkotlinx/coroutines/CoroutineExceptionHandlerKt$CoroutineExceptionHandler$1\n+ 2 TeamCompassViewModel.kt\ncom/example/teamcompass/ui/TeamCompassViewModel\n*L\n1#1,48:1\n78#2,2:49\n*E\n"})
public final class TeamCompassViewModel$special$$inlined$CoroutineExceptionHandler$1 extends AbstractCoroutineContextElement implements CoroutineExceptionHandler {
   public TeamCompassViewModel$special$$inlined$CoroutineExceptionHandler$1(CoroutineExceptionHandler.Key $super_call_param$1) {
      super((CoroutineContext.Key)$super_call_param$1);
   }

   public void handleException(CoroutineContext context, Throwable exception) {
      int var4 = 0;
      Log.e("TeamCompassVM", "Unhandled coroutine exception", exception);
   }
}

File: tools/recovery/vine_vm_all/TeamCompassViewModel$special$$inlined$CoroutineExceptionHandler$1.kt
package com.example.teamcompass.ui

import android.util.Log
import kotlin.coroutines.AbstractCoroutineContextElement
import kotlin.coroutines.CoroutineContext
import kotlin.jvm.internal.SourceDebugExtension
import kotlinx.coroutines.CoroutineExceptionHandler
import kotlinx.coroutines.CoroutineExceptionHandler.Key

@SourceDebugExtension(["SMAP\nCoroutineExceptionHandler.kt\nKotlin\n*S Kotlin\n*F\n+ 1 CoroutineExceptionHandler.kt\nkotlinx/coroutines/CoroutineExceptionHandlerKt$CoroutineExceptionHandler$1\n+ 2 TeamCompassViewModel.kt\ncom/example/teamcompass/ui/TeamCompassViewModel\n*L\n1#1,48:1\n78#2,2:49\n*E\n"])
public class `TeamCompassViewModel$special$$inlined$CoroutineExceptionHandler$1` : AbstractCoroutineContextElement, CoroutineExceptionHandler {
   fun `TeamCompassViewModel$special$$inlined$CoroutineExceptionHandler$1`(`$super_call_param$1`: Key) {
      super(`$super_call_param$1` as kotlin.coroutines.CoroutineContext.Key);
   }

   public open fun handleException(context: CoroutineContext, exception: Throwable) {
      Log.e("TeamCompassVM", "Unhandled coroutine exception", exception);
   }
}
